libs/component-sample/src/lib/feature/side-navigation-sample/side-navigation-sample.component.ts

```diff
@@ -183,9 +192,9 @@ export class SideNavigationSampleComponent implements AfterViewInit, OnInit {
   ];
 
 
-  public pageHeader: string;
-  public linkEvent = new BehaviorSubject<object>(null);
-  sideNavExpanded = true;
+  // public pageHeader: string;
+  // public linkEvent = new BehaviorSubject<object>(null);
+  // sideNavExpanded = true;
   ngOnInit() {
```

Comment:
delete -- ngOnInit is unused without unnecessary filterChange$

Reply:
Removed.